enterprise/security/src/main/java/org/neo4j/server/security/enterprise/auth/EnterpriseSecurityModule.java

```diff
@@ -91,6 +91,9 @@ public void setup( Dependencies dependencies ) throws KernelException
             );
         life.add( securityLog );
 
+        Boolean allowTokenCreate = config.get( SecuritySettings.allow_publisher_create_token );
```

Comment:
I'd prefer to see a lower-case `boolean` here, to make it clear that we really don't expect to see a `null` pop out of this setting. Also, it makes auto-unboxing only happen once.